cps/CP1/CP1.md

```diff
@@ -22,9 +22,54 @@ The next step is the funding process, which will be an on-chain Treasury Proposa
 | --- | --- | :---: |
 |5|Create an on-chain Treasury proposal|-|
 
+### TEMPLATE
+```
+Title: *RFC: [TITLE OF PROPOSAL]*
+
+Tags: *cp-1, month, year*
+```
+#### Body
```

Comment:
i think that will be better to write

#### Body of the RFC must contain: 

What do you think?

Reply:
Yes, I will add it.